pkg/crossplane/service_redis.go

```diff
@@ -44,6 +44,12 @@ func (rsb RedisServiceBinder) Deprovisionable(ctx context.Context) error {
 	return nil
 }
 
+// GetClusterName returns the cluster name of the instance
+func (instance Instance) GetClusterName() string {
+	labels := instance.Composite.GetLabels()
+	return labels["service.syn.tools/cluster"]
+}
```

Comment:
This method should be in `instance.go` not in `service_redis.go`.

Reply:
Moved the method